src/common/ModalDialog/ModalDialog.js

```diff
@@ -75,7 +75,7 @@ const ModalDialog = ({ className, title, buttons, children, dataset, onCloseRequ
                     Array.isArray(buttons) && buttons.length > 0 ?
                         <div className={styles['buttons-container']}>
                             {buttons.map(({ className, label, icon, props }, index) => (
-                                <Button title={label} {...props} key={index} className={classnames(className, styles['action-button'])}>
+                                <Button title={label} {...props} key={index} className={classnames(className, label === 'Uninstall' ? styles['uninstall-button'] : styles['action-button'])}>
```

Comment:
Shouldn't check for the label, use the className property or add a new property to the buttons array to apply style accordingly